python/ray/train/tests/test_new_persistence.py

```diff
@@ -575,15 +558,22 @@ def test_trainer(
         ├── artifact-rank=1-iter=1.txt
         └── ...
     """
-    LOCAL_CACHE_DIR = tmp_path / "ray_results"
-    monkeypatch.setenv("RAY_AIR_LOCAL_CACHE_DIR", str(LOCAL_CACHE_DIR))
```

Comment:
I patched the local dir before so that nothing got written to `~/ray_results`, but instead a temp directory that is unique per test. I didn't want the staging directories of multiple tests to be mixed.